sdk/search/Azure.Search.Documents/src/Indexes/Models/ListIndexStatsSummary.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Search.Documents.Indexes.Models
+{
+    public partial class ListIndexStatsSummary
```

Comment:
Had to add this file to force `ListIndexStatsSummary` being public.

Reply:
I don't understand why you need to explicitly make this type public. Since you have a public method that returns this type, my understanding is that the type should automatically be generated as public.

For example, the [AnalyzeText](https://github.com/Azure/azure-sdk-for-net/blob/e41317405bb80b4d59c98743d6800d69b6e11636/sdk/search/Azure.Search.Documents/src/Indexes/SearchIndexClient.cs#L367) method is public, so the `AnalyzedTokenInfo` return type is also generated as public. Let me know if I'm missing something here.

Reply:
Me neither, but it was being generated as `internal` without this file. Maybe I was doing something wrong? I'll try removing it and letting you know what happens.

Reply:
Just double checked and yeah, it's generated as an `internal` class, for some reason. The weird thing is that `IndexStatisticsSummary`, another class that's only related to this API, is `public` by default with no issues.

Reply:
I'll approve the PR, but to better understand why this is happening, could you check with the code generation team? Thanks!